src/models/Users.js

```diff
@@ -0,0 +1,33 @@
+import { observable, action, computed } from "mobx";
+
+export default class Users {
+      @observable id;
+      @observable avatar;
+      @observable firstName;
+      @observable lastName;
+      @observable email;
+      @observable login;
+      @observable officeId;
+      @observable isAdmin;
+
+      @observable devices = [];
+      addDevice(deviceId) {
+        devices.push(device)
+      }
+      removeDevice(deviceId) {
+
+      }
+
+      @observable places = [];
+
+      constructor({id,avatar,firstName,lastName,email,login,officeId,isAdmin}) {
+          this.id=id;
```

Comment:
It's better practice to use spaces for readability.
```this.id = id;```
```{ id, avatar }```
etc

I think ESlint should check this, not sure why it didn't work this time